tests/unit/serializers/new-platform-flexberry-g-i-s-layer-metadata-test.js

```diff
@@ -3,7 +3,19 @@ import { moduleForModel, test } from 'ember-qunit';
 moduleForModel('new-platform-flexberry-g-i-s-layer-metadata', 'Unit | Serializer | new-platform-flexberry-g-i-s-layer-metadata', {
   // Specify the other units that are required for this test.
   needs: [
-    'model:new-platform-flexberry-g-i-s-layer-metadata'
+    'serializer:new-platform-flexberry-g-i-s-layer-metadata',
```

Comment:
as I remember serializer tests needs only current model and serializer?

Reply:
Flexberry generator generates tests like so.